src/index.js

```diff
@@ -0,0 +1,84 @@
+import './style.css';
+import changeState from './js/changeState.js';
+import {
+  add, removeAllCompleted, edit, remove,
+} from './js/addRemove.js';
+
+// Call HTML Elements
+const mainListContainer = document.getElementById('list');
+const addIcon = document.getElementById('add-icon');
+const inputField = document.querySelector('.input-container');
+const clearSelectedBtn = document.getElementById('clear-selected');
+
```

Comment:
- Kindly provide an array of objects requested on your project requirements:
![Screen Shot 2021-11-02 at 6 30 39](https://user-images.githubusercontent.com/22258274/139846763-c1d96e26-fa46-4f3d-a232-d928646b959c.png)

**Example:**
```
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```

Reply:
Hi man, I honestly don't understand what you want me to do right here, so may you kindly example.

The funny thing about your review is that you approved my second PR without the array of objects requested you requested on this review